src/js/overlay.js

```diff
@@ -131,13 +131,16 @@ const Overlay = function(id, opts) {
 };
 
 Overlay.prototype.open = function() {
+	// Prevent page scroll for open modals or fullscreen overlays.
+	if (this.opts.modal || this.opts.fullscreen) {
+		this.originalOverflow = document.body.style.overflow;
+		document.body.style.overflow = 'hidden';
```

Comment:
Does this retain the original scroll position too?

Reply:
Yes :)